main-settings/src/main/scala/sbt/Structure.scala

```diff
@@ -99,10 +99,7 @@ sealed abstract class SettingKey[T]
 
   protected[this] def make[S](other: Initialize[S], source: SourcePosition)(
       f: (T, S) => T
-  ): Setting[T] = {
-    import TupleSyntax._
-    set((this, other)(f), source)
-  }
+  ): Setting[T] = set(this.zipWith(other)(f), source)
```

Comment:
`Initialize#zipWith` is like a specialized tuple syntax for Tuple2?

Reply:
Yep.  It's special-cased and wires directly to Apply, using the Tuple2 instance of AList.